.travis.yml

```diff
@@ -1,27 +1,20 @@
-language: node_js
+
+language: objective-c
+compiler: clang
 node_js: '0.10'
 env:
   global:
     - secure: kjz286d6rs2VHpv4WnxAOYXcdTIaTaABZ0SNXf27r5H+NDu5qx8ZSeCmj/9op/sDjTtpkLKhi2a0njUCWwak28RAWP3dNkT92MkUPVKBkevznSxWeqwLHoUos3AJnhPMB+cwrWvmRC+Ljt5K1q8SxQsS/HXM9pzKL3DG3qxM+eo=
-branches:
-  only:
-    - master
-notifications:
-  email:
-    - jonas@halfdanj.dk
+  matrix:
+    - NAME="OSX" BEFORE_THE_INSTALL="" SCRIPT_TO_RUN="scripts/osx/testTemplate.sh" AFTER_THE_SUCCESS="echo 'openFrameworks OSX Build Passing'"
+    - NAME="iOS" BEFORE_THE_INSTALL="" SCRIPT_TO_RUN="scripts/ios/testTemplate.sh" AFTER_THE_SUCCESS="echo 'openFrameworks iOS Build Passing'"
+    - NAME="Docs" BEFORE_THE_INSTALL="scripts/dev/documentation.sh before_install" SCRIPT_TO_RUN="scripts/dev/documentation.sh script" AFTER_THE_SUCCESS="scripts/dev/documentation.sh after_success"
 before_install:
-  - 'if [[ $TRAVIS_PULL_REQUEST == "false" ]]; then sudo apt-get install python-software-properties; fi'
-  - 'if [[ $TRAVIS_PULL_REQUEST == "false" ]]; then sudo apt-add-repository -y ppa:libreoffice/libreoffice-4-2; fi'
-  - 'if [[ $TRAVIS_PULL_REQUEST == "false" ]]; then sudo apt-get update; fi'
-  - 'if [[ $TRAVIS_PULL_REQUEST == "false" ]]; then sudo apt-get install doxygen; fi'
-  - 'if [[ $TRAVIS_PULL_REQUEST == "false" ]]; then sudo apt-get install ncftp; fi'
-  - 'if [[ $TRAVIS_PULL_REQUEST == "false" ]]; then gem install sass --version "=3.2.12"; fi'
-  - 'if [[ $TRAVIS_PULL_REQUEST == "false" ]]; then gem install compass --version "=0.12.2"; fi'
+    $BEFORE_THE_INSTALL
```

Comment:
I'm sorry I don't get what you are saying.

As another point, check out the "description" of the builds [here](https://travis-ci.org/danoli3/openFrameworks/builds/69404952) - the stuff after the box icon - it's not really useful to anyone looking at the build matrix, only having the names there would be much better (and meaningful)